README.rst

```diff
@@ -91,8 +91,6 @@ Main Website: https://certbot.eff.org
 
 Let's Encrypt Website: https://letsencrypt.org
 
-IRC Channel: #letsencrypt on `Freenode`_
```

Comment:
Can you remove the now unused link for Freenode as well? That line is:
```
 .. _Freenode: https://webchat.freenode.net?channels=%23letsencrypt
```

Reply:
Sure! Will do. I didn't realize that's what the `_` notation was doing :+1: